examples/ephemeral-spark/tamr-vm.tf

```
@@ -1,14 +1,12 @@
 module "tamr-vm" {
-  source = "git::git@github.com:Datatamer/terraform-emr-tamr-vm?ref=1.0.2"
+  source = "git::https://github.com/Datatamer/terraform-aws-tamr-vm.git?ref=4.1.0"
```

Not that this is wrong, but most of the other modules use the ssh uri instead of the http uri. I don't have a preference either way as long as we're consistent.
```suggestion
  source = "git::git@github.com/Datatamer/terraform-aws-tamr-vm.git?ref=4.1.0"
```